src/Features/ERDDAP/Platform/Observations/CurrentConditions/wind.tsx

```diff
@@ -45,10 +45,7 @@ export const WindCard: React.SFC<Props> = ({ datasets, unit_system }) => {
   if (speed.length > 0 && speed[0].readings.length > 0) {
     speedTitle =
       " - " +
-      round(
-        data_converter.convertTo(speed[0].readings[speed[0].readings.length - 1].reading, unit_system) as number,
-        1
-      ) +
+      round(data_converter.convertToNumber(speed[0].readings[speed[0].readings.length - 1].reading, unit_system), 1) +
```

Comment:
Codacy found an issue: [Identifier 'data_converter' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6089628)